edb/server/protocol/studio_ext.pyx

```diff
@@ -0,0 +1,107 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2019-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import base64
+import http
+import json
+import urllib.parse
+import os
+import mimetypes
+
+import immutables
+
+from edb import errors
+
+from edb.common import debug
+from edb.common import markup
+
+STATIC_FILES_DIR = 'build/studio'
```

Comment:
Use `get_shared_data_dir_path() / 'studio'` instead to make this work with production builds.